extensions/resteasy-classic/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/graal/ClientHttpEngineBuilder43Replacement.java

```diff
@@ -34,4 +36,10 @@ public ClientHttpEngineBuilder43Replacement resteasyClientBuilder(ResteasyClient
         }
         return this;
     }
+
+    /**
+     * Unused alias to implementation to implement the interface
```

Comment:
```suggestion
     * Unused alias to implement the {@link ClientHttpEngineBuilder} interface
```

Updating and force pushing this now.

Reply:
@geoand I just force pushed to fix this ^^

Reply:
👌